manifests/w/WaveMetrics/IgorPro9/9.05/WaveMetrics.IgorPro9.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS1.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WaveMetrics.IgorPro9
+PackageVersion: "9.05"
+PackageLocale: en-US
+Publisher: WaveMetrics
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: Igor Pro 9
+# PackageUrl:
+License: Copyright (c) WaveMetrics. All rights reserved.
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate